fjord/settings/base.py

```diff
@@ -627,6 +627,9 @@ def JINJA_CONFIG():
 
 ## Celery
 
+import djcelery
+djcelery.setup_loader()
+
```

Comment:
I think this should be in `fjord/settings/__init__.py` at the bottom instead of in `fjord/settings/base.py` in the middle.